skbio/diversity/alpha/_base.py

```diff
@@ -848,6 +850,51 @@ def osd(counts):
     return observed_otus(counts), singles(counts), doubles(counts)
 
 
+@experimental(as_of="0.4.0-dev")
+def pielou_e(counts):
+    r"""Calculate Pielou's Evenness index J'.
+
+    Pielou's Evenness is defined as:
+
+    .. math::
+
+       J' = \frac{(H)}{\ln(S)}
+
+    where :math:`H` is the Shannon-Wiener entropy of counts and :math:`S` is
+    the number of OTUs in the sample.
+
+    Parameters
+    ----------
+    counts : 1-D array_like, int
+        Vector of counts.
+
+    Returns
+    -------
+    double
+        Pielou's Evenness.
+
+    See Also
+    --------
+    shannon
+    heip_e
+
+    Notes
+    -----
+    The implementation here is based on the description given by Carlo Heip
```

Comment:
Can you add a reference to the original paper where this metric is introduced?

```
PIELOU, E. C., 1966. The measurement of diversity in different types of biological collections. Journal of Theoretical Biology, 13, 131-44.
```

I think the current wording is still correct (i.e., we are basing the implementation on Heip's paper and Wikipedia) because the original paper doesn't specify a logarithm base.